src/Dropdown.js

```diff
@@ -132,23 +132,32 @@ class Dropdown extends React.Component {
     const disabledClass = mapToCssModules('disabled', this.props.cssModule);
 
     const items = container.querySelectorAll(`.${menuClass} .${itemClass}:not(.${disabledClass})`);
+    const char = String.fromCharCode(e.which).toLowerCase();
+    const itemsOfChar = char !== '' &&
+      Array.from(items).filter(item =>
+        !!item.textContent && item.textContent.substr(0, 1).toLowerCase() === char);
 
     if (!items.length) return;
 
     let index = -1;
-    for (let i = 0; i < items.length; i += 1) {
-      if (items[i] === e.target) {
-        index = i;
-        break;
+
```

Comment:
If `char` is an empty string `itemsOfChar` will be `false` and this will throw an error.